### PR TITLE
figurine: Add 1.0.1

### PR DESCRIPTION
**Pull request by wenmin92**
**[figurine](https://github.com/arsham/figurine)**

Description: Print your name in style
License: Apache-2.0

Preview:
image

### CODE REVIEW

File: bucket/figurine.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "homepage": "https://github.com/arsham/figurine",
+    "description": "Print your name in style",
```
Comment (Ash258) [inline bucket/figurine.json]: https://github.com/arsham/rainbow#see-also

```suggestion
    "description": "Print any text using ASCII art in FIGlet.",
```

File: bucket/figurine.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "version": "1.0.1",
+    "license": "Apache-2.0",
+    "url": "https://github.com/arsham/figurine/releases/download/v1.0.1/figurine_windows_v1.0.1.zip",
+    "hash": "e9afc6b215c0971dbe08ff4bd13678c6e34b265d412e5595f6cb16919e932ce4",
+    "extract_dir": "deploy",
+    "bin": "figurine.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/arsham/figurine/releases/download/v$version/figurine_windows_v$version.zip"
```
Comment (Ash258) [inline bucket/figurine.json]:
```suggestion
        "url": "https://github.com/arsham/figurine/releases/download/v$version/figurine_windows_v$version.zip",
        "hash": {
            "url": "https://github.com/arsham/figurine/releases/tag/v$version"
        }
```

**Comment by wenmin92** [comment]
> Move this PR into main bucket

Criteria for including apps in the main bucket:
- a reasonably well-known and widely used developer tool

I thought it was not so well-known.

**Comment by wenmin92** [comment]
Has moved to main bucket.

**Pull request closed without merging**